randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -825,7 +828,17 @@ Extra - asset_id: collision_camera_012
               All of the following:
                   Gravity Suit
                   Space Jump or Simple IBJ
-              Spider Magnet and Use Spin Boost
+              All of the following:
+                  Spider Magnet
+                  Any of the following:
+                      Use Spin Boost
+                      Morph Ball and Single-wall Wall Jump (Beginner)
+              All of the following:
+                  # https://www.youtube.com/watch?v=SUxSv6YFK3Y
+                  Grapple Beam and Grapple Movement (Intermediate)
+                  Any of the following:
+                      Use Spin Boost
+                      Morph Ball and Single-wall Wall Jump (Beginner)
```

Comment:
Both of these branches could have an alternative for using Flash Shift in the Wall Jumps.